src/test/ReactDefaultPerf.js

```diff
@@ -213,6 +221,10 @@ var ReactDefaultPerf = {
             totalTime,
             Array.prototype.slice.call(args, 1)
           );
+
+          if (fnName === 'putListener') {
+            ReactDefaultPerf._prevEventListener = typeof isFunction;
```

Comment:
I'm skeptical this would work.  I think there is a single ReactDefaultPerf shared for all components; you are setting this field as a static field without regard to the current component.